www/app/App.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { App } from './App';
+import { renderWithIntl } from '../shared/test/utils';
+
+beforeAll(() => {
+    document.documentElement.innerHTML = '<div id="__next" style="position: static; top: auto; left: auto; right: auto" />';
+});
+
+const getTree = (props) => renderWithIntl(
+    <App
+        Component={ () => <div>Hello World</div> } // eslint-disable-line react/jsx-no-bind
+        { ...props } />,
+);
+
+it('should render correctly', () => {
+    const { container } = getTree();
+
+    expect(container).toHaveTextContent('Hello World');
+});
+
+it('should return entries on getInitialProps', async () => {
```

Comment:
What does entries mean? Perhaps a better test title would be: `should call page component getInitialProps`? However I don’t see the need to have this test as we don’t have any actual getInitialProps on the app.

Reply:
I agree, this test and the next one may be deleted. Thanks!